bili.config.dev.js

```diff
@@ -0,0 +1,14 @@
+import { name } from "./package.json";
+
+/** @type {import('bili').Config} */
```

Comment:
please remove this comment

Reply:
Do you understand that having it makes your code editor aware of config structure so it can autocomplete its properties and show documentation popups on hover?
Check out this: https://bili.egoist.sh/#/configuration-file#typescript

Reply:
I'm sorry but I don't like this. makes code looks weird and is not even clear what it does. I don't find it useful.